### PR TITLE
fix(typings): revert to previous typings + remove export of strava const

### PR DESCRIPTION
**Pull request by ntziolis**

### CODE REVIEW

File: index.d.ts [hunk 1]
```diff
@@ -155,6 +155,4 @@ export interface Strava {
     oauth: OAuthRoutes;
 }
 
-declare const strava : Strava;
```
Comment (nik0kin) [inline index.d.ts:158]: We still need this line (but without the extra space after `strava`, if you don't mind

Comment (ntziolis) [reply index.d.ts:158]: @nik0kin Are you saying you cannot export it properly without it? Or what do you mean when saying we need this line? 

This line says says that the js module has a NAMED export `strava`, which it does not. So the typing pretends there is a variable that if used is actually undefined during runtime. This is the issue we ran into when using this lib the first time.

If we keep the line in question:
`import { strava } from 'strava-v3`

But `strava` is actually defined afterwards. Are you saying on your end its actually defined when importing via the above way?

Comment (ntziolis) [reply index.d.ts:158]: Thank you for pointing this out. Fixed.

**Comment by ntziolis** [comment]
@nik0kin I also added an import syntax section in the readme.

**Comment by nik0kin** [comment]
I don't think we need the additional readme section. Or at least not so close to the top

1. This section explains how to use default imports with/without esModuleInterop, which isnt related to this strava library.

2. All the current examples are in commonjs import syntax. Maybe all the examples should be redone in esmodules syntax, with an additional example using a type/interface, but I think that should be done in a separate PR, so that typing can be fixed in the latest version

**Comment by ntziolis** [comment]
It cost us about half a day to figure out how to import this properly. So the section is to prevent others from running into this. Its pretty standard for libraries to have a section on top how to import a library correctly sue to different module formats being use etc.

**Pull request merged**